code/build.sbt

```diff
@@ -2,11 +2,12 @@ organization := "com.feth"
 
 name := "play-easymail"
 
+scalaVersion := "2.11.11"
```

Comment:
Intellij IDEA will not be able to compile properly without it. I'll file a bug there as it should probably pick up the first of `crossScalaVersions` if no explicit `scalaVersion` is set.